website/pages/api/newsletter-subscribe.ts

```diff
@@ -0,0 +1,49 @@
+import FormData from 'form-data';
+
+export default async function handler(req, res) {
```

Comment:
This is not going to work because we are doing `next export` as static website. 
If you want to add such thing, you should add the handler at https://github.com/the-guild-org/website/tree/master/packages/website-helper-worker

Reply:
oh ok, I didn't realize that, thanks for pointing that out